columntypes/columntypes.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package columntypes
```

Comment:
I see that the functions from this package are used only by the writer, and only in 1-2 places each. I'm wondering if we can simply pull these into the writer as unexported functions.

Reply:
Yep, I can move it into the writer package. But the reason why it's a separate package is that I'm going to add here some methods for the source.
So WDYT will it be better to move it to the writer package and in the feature add the same file for the source?

Reply:
Will the methods you'll use for the source be closely related to these methods?

Reply:
I think I will use the `GetColumnTypes` method in a source too.

Reply:
@hariso jfyi

I've moved these methods into the `writer` package.